app/assets/javascripts/alternate_catalog.js

```diff
@@ -3,18 +3,19 @@
     container: null,
     titleElement: null,
 
-    init: function(el) {
+    init: function (el) {
       this.container = $(el);
-      this.titleElement = this.container.find('.alternate-catalog-title');
+      this.titleElement = this.container.find(".alternate-catalog-title");
 
       // Insert between the 3rd and 4th document
       this.injectAlternateCatalogInotResults();
-
     },
 
-    injectAlternateCatalogInotResults: function() {
-      var $documents = $('#documents');
-      var afterThird = $documents.find('.document-position-2').after(this.container);
+    injectAlternateCatalogInotResults: function () {
+      var $documents = $("#documents");
+      var afterThird = $documents
+        .find(".document-position-3")
```

Comment:
The main change is switching from `.find(".document-position-2")` to `.find(".document-position-3")` since the counter used in Blacklight switched from starting at 0 to starting at 1. The rest of the changes are just linter recommendations.